site/docs/be/docker.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect: docs/be/docker.html
```

Comment:
Because I cleaned up these pages in #2561, and the underlying issue is #2556.

These pages were still available and served some time ago and appeared in our site search. I prefer to have redirects in case people have links pointing to it.